From 9c2ba09b0c0b5e0f686bfe2056cebad6f843e723 Mon Sep 17 00:00:00 2001
From: graysky <therealgraysky@proton.me>
Date: Sat, 20 Aug 2022 16:38:10 -0400
Subject: [PATCH 04/13] arch-nspawn-arm-fix

---
 src/makechrootpkg.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/makechrootpkg.in b/src/makechrootpkg.in
index b278695..ca7c554 100644
--- a/src/makechrootpkg.in
+++ b/src/makechrootpkg.in
@@ -251,7 +251,7 @@ download_sources() {
 	# Ensure sources are downloaded
 	sudo -u "$makepkg_user" --preserve-env=GNUPGHOME,SSH_AUTH_SOCK \
 		env SRCDEST="$SRCDEST" BUILDDIR="$WORKDIR" \
-		makepkg --config="$copydir/etc/makepkg.conf" --verifysource -o "${verifysource_args[@]}" ||
+    makepkg -A --config="$copydir/etc/makepkg.conf" --verifysource -o "${verifysource_args[@]}" ||
 		die "Could not download sources."
 }
 
-- 
2.44.0

