From a6930d64c956e0f01d46e232f0c8d6efb43671f0 Mon Sep 17 00:00:00 2001
From: graysky <therealgraysky@proton.me>
Date: Sat, 20 Aug 2022 16:38:23 -0400
Subject: [PATCH 05/12] makechrootpkg-no-default-logging

---
 src/makechrootpkg.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/makechrootpkg.in b/src/makechrootpkg.in
index d3ab2a1..a33219d 100644
--- a/src/makechrootpkg.in
+++ b/src/makechrootpkg.in
@@ -14,7 +14,7 @@ source /usr/share/makepkg/util/config.sh
 
 shopt -s nullglob
 
-default_makepkg_args=(--syncdeps --noconfirm --log --holdver --skipinteg)
+default_makepkg_args=(--syncdeps --noconfirm --holdver --skipinteg)
 makepkg_args=("${default_makepkg_args[@]}")
 verifysource_args=()
 chrootdir=
-- 
2.42.0

