From 4fe137431c7413bf0d4238d6f1ef0da03c934da7 Mon Sep 17 00:00:00 2001
From: graysky <graysky@archlinux.us>
Date: Tue, 25 May 2021 15:07:03 -0400
Subject: [PATCH 5/7] makechrootpkg: no default logging

---
 makechrootpkg.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/makechrootpkg.in b/makechrootpkg.in
index bfe8345..7470670 100644
--- a/makechrootpkg.in
+++ b/makechrootpkg.in
@@ -17,7 +17,7 @@ source /usr/share/makepkg/util/config.sh
 
 shopt -s nullglob
 
-default_makepkg_args=(--syncdeps --noconfirm --log --holdver --skipinteg)
+default_makepkg_args=(--syncdeps --noconfirm --holdver --skipinteg)
 makepkg_args=("${default_makepkg_args[@]}")
 verifysource_args=()
 chrootdir=
-- 
2.31.1

