From 0dcec2968af357adcaaa78f683945905fcd878f9 Mon Sep 17 00:00:00 2001
From: graysky <graysky@archlinux.us>
Date: Tue, 25 May 2021 15:11:20 -0400
Subject: [PATCH 6/9] archbuild: no setarch

---
 archbuild.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/archbuild.in b/archbuild.in
index 5b98976..b62794f 100644
--- a/archbuild.in
+++ b/archbuild.in
@@ -69,7 +69,7 @@ if ${clean_first} || [[ ! -d "${chroots}/${repo}-${arch}" ]]; then
 
 	rm -rf --one-file-system "${chroots}/${repo}-${arch}"
 	(umask 0022; mkdir -p "${chroots}/${repo}-${arch}")
-	setarch "${arch}" mkarchroot \
+	mkarchroot \
 		-C "${pacman_config}" \
 		-M "${makepkg_config}" \
 		"${chroots}/${repo}-${arch}/root" \
-- 
2.32.0

