From 3a202b549fbe44555f7ec2e7abf109bd7a3cb155 Mon Sep 17 00:00:00 2001
From: graysky <therealgraysky@proton.me>
Date: Sat, 20 Aug 2022 16:38:35 -0400
Subject: [PATCH 06/13] archbuild-no-setarch

---
 src/archbuild.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/archbuild.in b/src/archbuild.in
index 2f3faf9..00facd1 100644
--- a/src/archbuild.in
+++ b/src/archbuild.in
@@ -95,7 +95,7 @@ if ${clean_first} || [[ ! -d "${chroots}/${repo}-${arch}" ]]; then
 
 	rm -rf --one-file-system "${chroots}/${repo}-${arch}"
 	(umask 0022; mkdir -p "${chroots}/${repo}-${arch}")
-	setarch "${set_arch}" mkarchroot \
+	mkarchroot \
 		-C "${pacman_config}" \
 		-M "${makepkg_config}" \
 		"${chroots}/${repo}-${arch}/root" \
-- 
2.45.2

