From 44ac99f918a96e680c22072ca6182c233018e4b6 Mon Sep 17 00:00:00 2001
From: graysky <therealgraysky@proton.me>
Date: Sat, 20 Aug 2022 16:38:35 -0400
Subject: [PATCH 06/10] archbuild-no-setarch

---
 src/archbuild.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/archbuild.in b/src/archbuild.in
index 4d955bf6ab34..917ab29420c3 100644
--- a/src/archbuild.in
+++ b/src/archbuild.in
@@ -104,7 +104,7 @@ if ${clean_first} || [[ ! -d "${chroots}/${repo}-${arch}" ]]; then
 
 	rm -rf --one-file-system "${chroots}/${repo}-${arch}"
 	(umask 0022; mkdir -p "${chroots}/${repo}-${arch}")
-	setarch "${set_arch}" mkarchroot \
+	mkarchroot \
 		-C "${pacman_config}" \
 		-M "${makepkg_config}" \
 		"${chroots}/${repo}-${arch}/root" \
-- 
2.52.0

