From 77ea66d008c8950d85b508f5cf02d9453984912a Mon Sep 17 00:00:00 2001
From: graysky <therealgraysky@proton.me>
Date: Sat, 20 Aug 2022 16:38:46 -0400
Subject: [PATCH 07/13] makechrootpkg-don-t-delete-MAKEFLAGS-and-PACKAGER

---
 src/makechrootpkg.in | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/makechrootpkg.in b/src/makechrootpkg.in
index f2c72d3..bb21518 100644
--- a/src/makechrootpkg.in
+++ b/src/makechrootpkg.in
@@ -177,8 +177,7 @@ prepare_chroot() {
 
 	$install -d "$copydir"/{build,startdir,{pkg,srcpkg,src,log}dest}
 
-	sed -e '/^MAKEFLAGS=/d' -e '/^PACKAGER=/d' -i "$copydir/etc/makepkg.conf"
-	for x in BUILDDIR=/build PKGDEST=/pkgdest SRCPKGDEST=/srcpkgdest SRCDEST=/srcdest LOGDEST=/logdest \
+  for x in BUILDDIR=/build PKGDEST=/pkgdest SRCPKGDEST=/srcpkgdest SRCDEST=/srcdest LOGDEST=/logdest
 		"MAKEFLAGS='${MAKEFLAGS:-}'" "PACKAGER='${PACKAGER:-}'"
 	do
 		grep -q "^$x" "$copydir/etc/makepkg.conf" && continue
-- 
2.45.2

