From 07ceddf8a8bb22f64d038ff6d69b02b8ca1618a9 Mon Sep 17 00:00:00 2001
From: graysky <graysky@archlinux.us>
Date: Tue, 25 May 2021 15:11:33 -0400
Subject: [PATCH 7/9] makechrootpkg: don't delete MAKEFLAGS and PACKAGER

---
 makechrootpkg.in | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/makechrootpkg.in b/makechrootpkg.in
index 7470670..664bac1 100644
--- a/makechrootpkg.in
+++ b/makechrootpkg.in
@@ -172,9 +172,7 @@ prepare_chroot() {
 
 	$install -d "$copydir"/{build,startdir,{pkg,srcpkg,src,log}dest}
 
-	sed -e '/^MAKEFLAGS=/d' -e '/^PACKAGER=/d' -i "$copydir/etc/makepkg.conf"
-	for x in BUILDDIR=/build PKGDEST=/pkgdest SRCPKGDEST=/srcpkgdest SRCDEST=/srcdest LOGDEST=/logdest \
-		"MAKEFLAGS='${MAKEFLAGS:-}'" "PACKAGER='${PACKAGER:-}'"
+	for x in BUILDDIR=/build PKGDEST=/pkgdest SRCPKGDEST=/srcpkgdest SRCDEST=/srcdest LOGDEST=/logdest
 	do
 		grep -q "^$x" "$copydir/etc/makepkg.conf" && continue
 		echo "$x" >>"$copydir/etc/makepkg.conf"
-- 
2.32.0

