From 75db3b5ee86f52f0b99b8244bb2663cb321b3c85 Mon Sep 17 00:00:00 2001
From: graysky <therealgraysky@proton.me>
Date: Sat, 20 Aug 2022 16:39:05 -0400
Subject: [PATCH 08/13] makechrootpkg-gotmpdir

---
 src/makechrootpkg.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/makechrootpkg.in b/src/makechrootpkg.in
index bb21518..eaa66b8 100644
--- a/src/makechrootpkg.in
+++ b/src/makechrootpkg.in
@@ -231,7 +231,7 @@ _chrootbuild() {
 	sudo --preserve-env=SOURCE_DATE_EPOCH \
 		--preserve-env=BUILDTOOL \
 		--preserve-env=BUILDTOOLVER \
-    -iu builduser DISTCC_IO_TIMEOUT=1200 DISTCC_DIR='/build/.distcc' bash -c 'cd /startdir; makepkg "$@"' -bash "$@"
+    -iu builduser DISTCC_IO_TIMEOUT=1200 DISTCC_DIR='/build/.distcc' GOTMPDIR='/build' bash -c 'cd /startdir; makepkg "$@"' -bash "$@"
 	ret=$?
 	case $ret in
 		0|14)
-- 
2.45.2

